lib/omnibus/packagers/ips.rb

```diff
@@ -0,0 +1,191 @@
+#
+# Copyright 2016 Chef Software, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+module Omnibus
+  class Packager::IPS < Packager::Base
+    id :ips
+
+    setup do
+      # Copy the full-stack installer into our scratch directory, accounting for
+      # any excluded files.
+      #
+      # /opt/hamlet => /tmp/daj29013/proto_install/opt/hamlet
+      # Create the proto_install directory inside staging_dir
+      Dir.mkdir("#{staging_dir}/proto_install")
+      destination = File.join(staging_path('proto_install'), project.install_dir)
+      FileSyncer.sync(project.install_dir, destination, exclude: exclusions)
+      create_transform_file
+    end
+
+    build do
+      build_me
+    end
+
+    def build_me
+      generate_pkg_manifest
+      create_ips_repo
+      publish_ips_pkg
+      view_repo_info
+      publish_as_pkg_archive
+    end
+
+    #
+    # Generate package manifest
+    #
+    # Package manifest is generally divided into three different parts:
+    # 1) Package metadata
+    # 2) Package contents
+    # 3) Package dependencies
+    #
+    def generate_pkg_manifest
+      generate_pkg_metadata
+      generate_pkg_contents
+      generate_pkg_deps
+    # Let's check the manifest and make sure all is right.
+      check_pkg_manifest
+    end
+
+    # Generate package metadata
```

Comment:
Need a `#` on the preceding line for parity to match the other yarddoc blocks.  (And because not having it makes me twitch.)